packages/server/src/lib/gost-aws.js

```diff
@@ -63,6 +92,7 @@ function send(message) {
         .catch((err) => console.error('Error sending SES email:', err, err.stack));
 }
 
-module.exports = { send };
-
-/*                                  *  *  *                                   */
+module.exports = {
+    getS3Client,
+    send,
```

Comment:
I think it would be useful to rename this function and its exported name to something a bit more verbose, like `sendEmail`.